app/views/shared/_field_with_error.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  label ||= ''
+  type ||= :text_field
+  qa ||= ''
+  select_options ||= []
+  input_options = {
+    class: type == :select ? "govuk-select" : "govuk-input govuk-!-width-full",
+    aria: {
+      describedby: @errors[field] ? "#{field.to_s}-error" : ''
+    },
+    data: {
+      qa: qa
+    }
+  }
+%>
+
+<div class="govuk-form-group<%= " govuk-form-group--error" if @errors[field]&.any? %>">
+  <% if label.blank? %>
+    <%= f.label field, class: "govuk-label" %>
```

Comment:
I wonder if we should be explicit about this Rails behaviour, ie:
```
label ||= field.humanize
```

Reply:
`.to_s.humanize` probably 👍